src/update-git-refs.ts

```diff
@@ -79,6 +80,31 @@ function findTrackables(doc: yaml.Document.Parsed): Trackable[] {
       'ref',
     );
 
+    let maybeDockerCommit: string | null = null;
+
+    // We only care about using the potential docker commit sha if we are using the global tracker
+    // as opposed to defining a tracker for the git config by itself. We don't need to care if no track is present because we will not update those anyway
+    if (
+      value.has('dockerImage') &&
+      !getStringValue(gitConfigBlock, 'trackMutableRef')
```

Comment:
I think we don't necessarily need this check.  It's hard to think about how it overlaps with `global` anyway. The idea of this change is it never actually affects the contents of the chart that's read, just a heuristic about which of several equivalent things to write, so I think this check is overkill...

Reply:
(ok global is a red herring, we don't process tracks in global. but i think it's ok to ignore anyway)